### ISSUE TITLE
Failed converting wav2vec2 model

### ISSUE THREAD

**Issue by piuy11** [aws-neuron/aws-neuron-sdk#628]
Hi,
I tried to convert [facebook/wav2vec2-large-960h-lv60-self](https://huggingface.co/facebook/wav2vec2-large-960h-lv60-self) model, but got following result.
Input sizes with (1,16000), (1,32000), (1,48000) succeded, but had problem over them.

Since I got same error message with Ubuntu & A100 GPU, it seems not to be a machine problem.

## Environment
- Instance type: Inf1.2xlarge
- OS: Amazon Linux 2

## Code to reproduce
```
import torch
import torch.neuron
from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor

model = Wav2Vec2ForCTC.from_pretrained("facebook/wav2vec2-large-960h-lv60-self")
model.eval()

input_size = (1, 64000)
input_audio = torch.rand(input_size)
model_neuron = torch.neuron.trace(model, example_inputs=[input_audio], strict=False)
model_neuron.save(f"stt_model_64000.pt")
```
## Result
```
Some weights of Wav2Vec2ForCTC were not initialized from the model checkpoint at facebook/wav2vec2-large-960h-lv60-self and are newly initialized: ['wav2vec2.masked_spec_embed']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
/home/ec2-user/.local/lib/python3.7/site-packages/transformers/models/wav2vec2/modeling_wav2vec2.py:583: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if attn_weights.size() != (bsz * self.num_heads, tgt_len, src_len):
/home/ec2-user/.local/lib/python3.7/site-packages/transformers/models/wav2vec2/modeling_wav2vec2.py:622: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if attn_output.size() != (bsz * self.num_heads, tgt_len, self.head_dim):
INFO:Neuron:All operators are compiled by neuron-cc (this does not guarantee that neuron-cc will successfully compile)
INFO:Neuron:Number of arithmetic operators (pre-compilation) before = 1230, fused = 1230, percent fused = 100.0%
WARNING:tensorflow:From /home/ec2-user/.local/lib/python3.7/site-packages/torch_neuron/ops/aten.py:2277: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
INFO:Neuron:Compiling function _NeuronGraph$1217 with neuron-cc
INFO:Neuron:Compiling with command line: '/home/ec2-user/.local/bin/neuron-cc compile /tmp/tmplsb3ocj_/graph_def.pb --framework TENSORFLOW --pipeline compile SaveTemps --output /tmp/tmplsb3ocj_/graph_def.neff --io-config {"inputs": {"tensor.1:0": [[1, 64000], "float32"]}, "outputs": ["Linear_5/aten_linear/Add:0"]} --verbose 35'
.[08:44:19] /opt/brazil-pkg-cache/packages/DmlcTvm/DmlcTvm-1.13.0.0/AL2_x86_64/generic-flavor/src/src/relay/pass/do_replication.cc:66: Replication is not applied to group convolution. Kernel and data input channels do not match[1, 128, 64, 1024]  [1, 1, 327, 1024]
..............02/01/2023 08:49:01 AM ERROR [Stargazer]: File network.cpp:835, Assertion 'sbuf.qTpbWriteAccessCheck(sbAddress, numBytes)' failed: Bad SB write address alignment for Load t11035_transpose_11271-vdns-14143_i0_TO_Wav2Vec2Model_3/Wav2Vec2FeatureEncoder_3/Wav2Vec2LayerNormConvLayer_21/LayerNorm_7/aten_layer_norm/batchnorm/mul_2_t27493_i15_Load_I-27494-15: 92306

INFO:Neuron:Compile command returned: 1
WARNING:Neuron:torch.neuron.trace failed on _NeuronGraph$1217; falling back to native python function call
ERROR:Neuron:neuron-cc failed with the following command line call:
/home/ec2-user/.local/bin/neuron-cc compile /tmp/tmplsb3ocj_/graph_def.pb --framework TENSORFLOW --pipeline compile SaveTemps --output /tmp/tmplsb3ocj_/graph_def.neff --io-config '{"inputs": {"tensor.1:0": [[1, 64000], "float32"]}, "outputs": ["Linear_5/aten_linear/Add:0"]}' --verbose 35
Traceback (most recent call last):
  File "/home/ec2-user/.local/lib/python3.7/site-packages/torch_neuron/convert.py", line 392, in op_converter
    item, inputs, compiler_workdir=sg_workdir, **kwargs)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/torch_neuron/decorators.py", line 229, in trace
    'neuron-cc failed with the following command line call:\n{}'.format(command))
subprocess.SubprocessError: neuron-cc failed with the following command line call:
/home/ec2-user/.local/bin/neuron-cc compile /tmp/tmplsb3ocj_/graph_def.pb --framework TENSORFLOW --pipeline compile SaveTemps --output /tmp/tmplsb3ocj_/graph_def.neff --io-config '{"inputs": {"tensor.1:0": [[1, 64000], "float32"]}, "outputs": ["Linear_5/aten_linear/Add:0"]}' --verbose 35
INFO:Neuron:Number of arithmetic operators (post-compilation) before = 1230, compiled = 0, percent compiled = 0.0%
INFO:Neuron:The neuron partitioner created 1 sub-graphs
INFO:Neuron:Neuron successfully compiled 0 sub-graphs, Total fused subgraphs = 1, Percent of model sub-graphs successfully compiled = 0.0%
INFO:Neuron:Compiled these operators (and operator counts) to Neuron:
INFO:Neuron:Not compiled operators (and operator counts) to Neuron:
INFO:Neuron: => aten::Int: 264 [supported]
INFO:Neuron: => aten::_convolution: 8 [supported]
INFO:Neuron: => aten::_weight_norm: 1 [supported]
INFO:Neuron: => aten::add: 49 [supported]
INFO:Neuron: => aten::bmm: 48 [supported]
INFO:Neuron: => aten::contiguous: 72 [supported]
INFO:Neuron: => aten::dropout: 99 [supported]
INFO:Neuron: => aten::gelu: 32 [supported]
INFO:Neuron: => aten::layer_norm: 57 [supported]
INFO:Neuron: => aten::linear: 146 [supported]
INFO:Neuron: => aten::mul: 48 [supported]
INFO:Neuron: => aten::reshape: 24 [supported]
INFO:Neuron: => aten::size: 48 [supported]
INFO:Neuron: => aten::slice: 4 [supported]
INFO:Neuron: => aten::softmax: 24 [supported]
INFO:Neuron: => aten::transpose: 137 [supported]
INFO:Neuron: => aten::unsqueeze: 1 [supported]
INFO:Neuron: => aten::view: 168 [supported]
Traceback (most recent call last):
  File "generate.py", line 10, in <module>
    model_neuron = torch.neuron.trace(model, example_inputs=[input_audio], strict=False)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/torch_neuron/convert.py", line 195, in trace
    cu.stats_post_compiler(neuron_graph)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/torch_neuron/convert.py", line 503, in stats_post_compiler
    "No operations were successfully partitioned and compiled to neuron for this model - aborting trace!")
RuntimeError: No operations were successfully partitioned and compiled to neuron for this model - aborting trace!
```

**Comment by shebbur-aws** [comment]
Thank you for reporting the issue. We are currently working on reproducing the issue on our end.

Regards,
Shruthi.

**Comment by shebbur-aws** [comment]
We are able to repro the issue and debugging it currently. Will post once we have identified the root cause and fix.

**Comment by jeffhataws** [comment]
Hi @piuy11 , we have implemented a fix which will be available in a future release. We will update you when that happens.

**Comment by mrnikwaws** [comment]
Hi @piuy11,

The problem you reported has been resolved with the 2.9.0 release of the Neuron SDK (https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html).  Please update your test environment with the new packages and retry.

I am going to close this issue, however if the problem is still occurring please re-open it.
